demo/components/presentational/SellOrdersList.test.js

```diff
@@ -4,8 +4,12 @@ import SellOrdersList from "./SellOrdersList";
 
 describe("SellOrdersList", () => {
   it("renders the component", async () => {
+    const renderItem = () => {};
```

Comment:
In the future this function could maybe use a clearer name. Not going to gate merging.

Reply:
Is `itemRenderer` a good name?

Reply:
I like that better, yes. I think the best might be ____Renderer where you say what the item is instead of using a generic word like `item`